packages/itwin/measure-tools/src/measurements/DistanceMeasurement.ts

```diff
@@ -51,6 +51,7 @@ export interface DistanceMeasurementProps extends MeasurementProps {
   startPoint: XYZProps;
   endPoint: XYZProps;
   showAxes?: boolean;
+  ratio?: number;
```

Comment:
I don't like such a generic term here called `ratio`  when the measurement calls it `sheetToWorldScale`. Also, why wouldn't the `sheetViewId` not be saved?

I think this information should be promoted to be a base class property (like an object property, that if defined, then has all the sheet metadata on it). Presumably other measurements may be supported in the sheet view, and there should be a standard way of handling that implementation.

Reply:
I've made some modifications to have the sheet metadata in the base measurement class, please verify if that's sufficient.
It's not an object but I did not feel like there was enough to justify an object for now.